github-issues/gh-issues.batch.yml

```
@@ -0,0 +1,32 @@
+name: refactor-go-gh-issue
+description: |
+  This campaign creates Jira tickets for each repository where fmt.Sprintf is replaced by strconv.Itoa.
+  It also keeps the tickets updated when the campaigns is re-applied.
+on:
+  - repository: github.com/sourcegraph-testing/tiny-go-test-repository
+
+steps:
+  - run: comby -in-place 'fmt.Sprintf("%d", :[v])' 'strconv.Itoa(:[v])' .go -matcher .go -exclude-dir .,vendor; echo "issue"
```

```suggestion
  - run: comby -in-place 'fmt.Sprintf("%d", :[v])' 'strconv.Itoa(:[v])' ${{ join repository.search_result_paths " " }}
```

If you use templating here you don't need the `exclude-dir` stuff etc.